server/src/main/scala/actors/DrtSystem.scala

```diff
@@ -62,7 +62,11 @@ trait DrtSystemInterface extends UserRoleProviderLike {
 }
 
 object DrtStaticParameters {
-  val expireAfterMillis: MillisSinceEpoch = 2 * oneDayMillis
+  val fortyEightHoursMillis: MillisSinceEpoch = 2 * oneDayMillis
+
+  def expire48HoursAgo(now: () => SDateLike): () => SDateLike = () => now().addDays(-2)
```

Comment:
this function name implies a verb (cause this to expire after 48 hours), however it is just just a variable.

Reply:
time48HoursAgo might be better?